### ISSUE TITLE
plotter.add_bounds_axes()

### ISSUE THREAD

**Issue by Jonas231** [pyvista/pyvistaqt#103]
Hello, 
I upgraded pyvista from 0.26.1 to 0.29 and pyvistaqt from 0.2 to 0.3.
Now I cannot set "plotter.add_bounds_axes()" in pyvistaqt anymore.

AttributeError: BackgroundPlotter has no attribute named add_bounds_axes

How can I add "add_bounds_axes()" again in my plot?

Kind regards,
Jonas

**Comment by Jonas231** [comment]
Okay. I just rediscovered a workaround:
plotter.show_bounds(font_size = 20, color='red')

**Comment by GuillaumeFavelier** [comment]
Indeed, since roughly `pyvista 0.27.0`, the function `add_bounds_axes()` has been changed to `show_bounds()` :)

For more infos on this, you can take a look at https://github.com/pyvista/pyvista/pull/141

Feel free to close if everthing works for you now :+1:
